client/components/thank-you/types.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { TranslateResult } from 'i18n-calypso';
+
+export type ThankYouNextStepProps = {
+	stepCta: React.ReactNode | React.ReactFragment;
```

Comment:
Might this be better expressed as a `TranslateResult`?

Reply:
I don't understand this change... Could you clarify?

Reply:
Sorry! That was a bit vague. I meant that it might be clearer to indicate that `stepCta` should be a `TranslateResult` -- we're expecting a value from `translate`.
```suggestion
	stepCta: TranslateResult;
```

Reply:
I don't think steCta is value from `translate` it's a _call to action_. Reviewing the code it's always a component, isn't it? I might be wrong but I can't see anywhere that this _stepCta_ is a translated string

Reply:
You're right! No need to change this then!